test/mysql_tests.erl

```diff
@@ -279,6 +280,53 @@ basic_queries(Pid) ->
 
     ok.
 
+filtermap_queries(Pid) ->
+    ok = mysql:query(Pid, ?create_table_t),
+    ok = mysql:query(Pid, <<"INSERT INTO t (id, tx) VALUES (1, 'text 1')">>),
+    ok = mysql:query(Pid, <<"INSERT INTO t (id, tx) VALUES (2, 'text 2')">>),
+    ok = mysql:query(Pid, <<"INSERT INTO t (id, tx) VALUES (3, 'text 3')">>),
```

Comment:
Just in case you didn't know you can do this in SQL:
```SQL
INSERT INTO t (id, tx) VALUES (1, 'text 1'), (2, 'text 2'), (3, 'text 3');
```

Reply:
I know ;) That's more or less an artifact there, I wasn't sure how many values I'd need, so just duplicated the line a few times over. I can change it to the `(...), (...), (...)`style if you like that better.

Reply:
No, it's not important.

Reply:
Sure, I'll merge it tonight or tomorrow.